### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kiranchitturi**
* Re-organize the spark-solr README
* Move the examples to a different folder

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,343 +1,247 @@
...
+import com.lucidworks.spark.rdd.SolrRDD
+val solrRDD = new SolrRDD(zkHost, collectionName, sc)
 
-Tweets will start flowing into Solr; be sure to let the streaming job run for a few minutes to build up a few thousand tweets in your socialdata collection. You can kill the job using ctrl-C.
+SolrRDD is an RDD of https://lucene.apache.org/solr/5_4_1/solr-solrj/org/apache/solr/common/SolrDocument.html[SolrDocument]
 
-[NOTE]
-====
-The sample command above includes properties for Twitter API credentials which need to be provided by you.
+==== via RDD ([line-through]#Java#)
```
Comment (ctargett) [inline README.adoc]: Why does this have a line-through? It's strange to have 2 headings that are the same except one word that is struck out, without any explanation for why.

Comment (kiranchitturi) [inline README.adoc]: Not sure about the line-through. I did not add it explicitly. May be my editor(IntelliJ) did. Will get rid of it

File: README.adoc [hunk 2]
```diff
@@ -1,343 +1,247 @@
...
- --class com.lucidworks.spark.SparkApp \
- ./target/spark-solr-1.0-SNAPSHOT-shaded.jar \
- twitter-to-solr -zkHost localhost:9983 -collection socialdata
+val options = Map(
+  "collection" -> "{solr_collection_name}",
+  "zkhost" -> "{zk_connect_string}"
+)
+val df = sqlContext.read.format("solr")
+  .options(options)
+  .load
```
Comment (ctargett) [inline README.adoc]: Is this snippet Scala? If not, what is it?

Comment (kiranchitturi) [inline README.adoc]: yes, Scala

**Comment by kiranchitturi** [comment]
Removed the line-through and added language to the source code snippets

**Comment by ctargett** [comment]
Looks good, Kiran. +1

**Pull request merged**